templates/vue/cypress/fixtures/three-nodes.json

```diff
@@ -240,5 +240,6 @@
       "target": 1531
     }
   ],
-  "groups": []
+  "groups": [],
+  "site-url": "http://localhost:8000"
```

Comment:
Is it possible to move this somewhere else so we don't have to repeat it in every fixture?

Reply:
I had to add this to every fixture because #682 requires imported Tapestry JSON files to contain the property. It was only missing because the tapestries were made prior to #682 - we wouldn't have to add it if we use newer Tapestries.

I might know a way to modify the Cypress code so this may not be necessary, but would require changing every single spec file that uses `cy.setup` - not sure if that's worthwhile.

Reply:
Ok I see, that's fine then. We can leave it as-is for now.